PhysicsTools/NanoAOD/plugins/LHETablesProducer.cc

```diff
@@ -66,11 +66,21 @@ class LHETablesProducer : public edm::global::EDProducer<> {
     std::vector<int> vals_pid;
     std::vector<int> vals_status;
     std::vector<int> vals_spin;
+    std::vector<Short_t> vals_firstMotherIdx;
+    std::vector<Short_t> vals_lastMotherIdx;
```

Comment:
Better to use `int16_t` instead of `Short_t` to be more explicit about the type.

Reply:
Hi @hqucms, I agree that int16_t is more explicit; actually Short_t comes from the fact that all *Idx varabiles are reported as Short_t in the [autoDoc](https://cms-nanoaod-integration.web.cern.ch/autoDoc/NanoAODv14/2024Prompt/doc_EGamma1_Run2024D-PromptReco-v1.html)
I suppose that autoDoc reports variables as Short_t even if they are defined as int16_t in the code.